docs/general/builtins/scalar-functions.rst

```diff
@@ -3214,6 +3214,29 @@ objects::
         SELECT 1 row in set (... sec)
 
 
+.. _scalar-null-or-empty:
+
+
+``null_or_empty(object)``
```

Comment:
Not sure if we could call this `empty()` instead ?

Reply:
I think `null_or_empty` is more explicit.
I'd expect `empty()` to return `null` for `null` as input.

Reply:
We would not be able to have something like `IS NULL_OR_EMPTY` right? it may be a bit counterintuitive than applying a function we get better index utilization.

Reply:
> We would not be able to have something like IS NULL_OR_EMPTY right? 

I don't think we should be adding special syntax for this. It's more complex as it requires parser and analyzer changes and it would make it way harder to use with ORMs.